src/pages.ts

```diff
@@ -1,7 +1,7 @@
 import {Redirects} from "..";
 
 const redirects: Redirects = {
-  "/": {redirect: false},
+  "/": {redirect: undefined},
```

Comment:
What is the benefit of switching from false to undefined? Or what is the logic behind this choice?